hamlpy/test/test_compiler.py

```diff
@@ -44,6 +44,12 @@ def test_attribute_dictionaries(self):
         # attribute whitespace is ignored
         self._test('%form{ id : "myform" }', "<form id='myform'></form>")
 
+        # HTML style
+        self._test('%form(foo=bar id="myform")', "<form foo='{{ bar }}' id='myform'></form>")
+
+        # multiple dicts
+        self._test('%a(a="b"){:c => "d"} Stuff', "<a a='b' c='d'>Stuff</a>")
```

Comment:
I can think of no good use for it but also no reason to not support it since it's in the spec.. no?